.github/workflows/deploy-production.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy AWS Production
+

```

Comment:
I've been adding this to include the branch name in the name of the run:

`run-name: Deploy ${{ github.ref }} to Production`